test/uk/gov/hmrc/gform/services/FormatValidationSpec.scala

```diff
@@ -184,6 +184,29 @@ class FormatValidationSpec extends Spec with GraphSpec {
     CompanyRegistrationNumber,
     "sample label is not a valid Company Registration Number")
 
+  "EORI" should "return valid EORI with 12 digits" in createSuccessTest(
```

Comment:
Also test that these are invalid: XGB123456789123, 3GB123456789123, XFR123456789123456 & 3FR123456789123456

Reply:
Please see latest commit @lutnos